docs/core/compatibility/serialization/6.0/jsonnumberhandlingattribute-behavior.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Breaking change: JsonNumberHandlingAttribute on non-number collection properties"
+description: Learn about the .NET 6 breaking change where JsonNumberHandlingAttribute can now only be applied to properties that are collections of numbers.
+ms.date: 11/05/2021
+---
+# JsonNumberHandlingAttribute on collection properties
+
+A minor breaking change was introduced in .NET 6 with regard to the <xref:System.Text.Json.Serialization.JsonNumberHandlingAttribute> attribute. If you apply the attribute to a property that's a collection of non-number values and attempt to serialize or deserialize the property, an <xref:System.InvalidOperationException> is thrown. The attribute is only valid for properties that are collections of number types, for example:
```

Comment:
```suggestion
A breaking change was introduced in .NET 6 with regard to the <xref:System.Text.Json.Serialization.JsonNumberHandlingAttribute> attribute. If you apply the attribute to a property that's a collection of non-number values and attempt to serialize or deserialize the property, an <xref:System.InvalidOperationException> is thrown. The attribute is only valid for properties that are collections of number types, for example:
```

I'm not sure it's a good idea to classify this as a "minor" breaking change. Any one who had code in place before that used this attribute in this way, would now experience an exception. It's still a breaking change, even though it may not affect as many consumers.